src/screens/TaskScreen.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export default function TaskScreen() {
+  return (
+    <View>
+      <Text>Welcome to the task screen!</Text>
+    </View>
+  );
+}
```

Comment:
good to see the connection is harnessed. I'll substitute in my current TaskScreen after this is merged to `main`.